examples/Google.Cloud.Functions.Examples.AdvancedDependencyInjection/Function.cs

```diff
@@ -72,14 +65,25 @@ public class Startup : FunctionsStartup
     }
 
     /// <summary>
+    /// <para>
     /// The actual Cloud Function using the dependencies.
     /// This uses both IOperationSingleton and IOperationScoped, and outputs the OperationId
     /// from each of them in HandleAsync. The service configuration provided in Startup
     /// means that for any given server, the SingletonOperationId property remains the same, but
     /// ScopedOperationId varies for each request.
     /// (Restarting the server will result in a different SingletonOperationId - nothing is persisted
     /// here.)
+    /// </para>
+    /// <para>
+    /// The FunctionsStartup attribute specifies the startup class to load. If you have multiple
```

Comment:
nit: `<see ref=FunctionsStartupAttribute />`?

Reply:
I've deliberately not done that because it makes it harder to read just in GitHub, which is where I'm expecting it to be read more than in VS. A VS user can always click through via the attribute itself if necessary.